durastore/src/main/java/org/duracloud/durastore/rest/AuxRest.java

```diff
@@ -42,16 +41,22 @@
 public class AuxRest extends BaseRest {
     private final Logger log = LoggerFactory.getLogger(AuxRest.class);
 
-    private RestUtil restUtil;
-
-    private StringDataStore dataStore;
+    private StringDataStoreFactory stringDataStoreFactory;
 
     @Inject
     private StorageAccountManager storageAccountManager;
 
-    @Autowired
-    public AuxRest(RestUtil restUtil) {
-        this.restUtil = restUtil;
+    public AuxRest() {
+        this(new StringDataStoreFactory());
+    }
+
+    protected AuxRest(StringDataStoreFactory stringDataStoreFactory) {
```

Comment:
Why not just pass in a StringDataStore here and omit the StringDataStoreFactory completely? I'm not seeing that the factory is providing much value.